lib/workload/stateless/metadata_manager/src/metadata_manager/wsgi.py

```diff
@@ -0,0 +1,7 @@
+import os
+
+from django.core.wsgi import get_wsgi_application
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "metadata_manager.settings.base")
```

Comment:
I think we removed the default Django setting assuming it's already set on the ENV.

Reply:
Yup, skel updates with #31